DEVELOPMENT.md

```diff
@@ -27,6 +27,43 @@
   ]
 ```
 
+### Testing GitHub actions on forks
+
+The lifecycle release process involves chaining a series of GitHub actions together such that:
+* The "build" workflow creates the artifacts
+  * .tgz files containing the lifecycle binaries, shasums for the .tgz files, a cosign public key, an SBOM, etc.
+  * OCI images containing the lifecycle binaries, tagged with their commit sha (for more information, see RELEASE.md)
+* The "draft-release" workflow finds the artifacts and downloads them, creating the draft release
+* The "post-release" workflow re-tags the OCI images that were created during the "build" workflow with the release version
+
+It can be rather cumbersome to test changes to these workflows, as they are heavily intertwined. Thus we recommend forking the buildpacks/lifecycle repository in GitHub and running through the entire release process end-to-end.
```

Comment:
I had good luck just updating image refs to points to ghcr.io/$gh-username/lifecycle, and changing the docker-login steps to login using `${{ secrets.GITHUB_TOKEN }}`. This meant I didn't have to set the `DOCKER_*` secrets, which was kinda nice.

Reply:
Thanks for the suggestion! I updated the script to accommodate this use case.